torch/functional.py

```diff
@@ -140,6 +141,80 @@ def btriunpack(LU_data, LU_pivots, unpack_data=True, unpack_pivots=True):
     return P, L, U
 
 
+def einsum(equation, *operands):
```

Comment:
Hmm it seems unfortunate that we have to move einsum into torch.nn.functional.einsum to use the varargs feature.